extension/js/common/platform/util.ts

```diff
@@ -32,3 +34,11 @@ export const moveElementInArray = <T>(arr: Array<T>, oldIndex: number, newIndex:
   arr.splice(newIndex, 0, arr.splice(oldIndex, 1)[0]);
   return arr;
 };
+
+export const iso2022jpToUtf = (content: Buf) => {
+  if (!TextDecoder) {
+    throw new Error('iso2022jpToUtf not implemented on node.');
+  }
+  const decoder = new TextDecoder('utf-8');
```

Comment:
Shouldn't it say `iso-2022-jp`?

Reply:
Yes, it should. I originally thought like you but that code was giving me a broken result.
I just researched it a little more and it's working now but I don't understand why.
Let me explain to you what I did:
Decoding result is broken:
```TS
const buf = Buf.fromUint8(node.content);
const decoder = new TextDecoder('iso-2022-jp');
return decoder.decode(content);
```
Decoding result is good:
```TS
const buf = Buf.fromRawBytesStr(node.rawContent!)
const decoder = new TextDecoder('iso-2022-jp');
return decoder.decode(content);
```

The difference is how I get `buf`. In the first example, I'm getting using `Buf.fromUint8(node.content);`, in the second `Buf.fromRawBytesStr(node.rawContent!)`.
But I don't understand why this affects decoding, I expected the same results.

Reply:
`rawContent` is my own addition to mime-decoder.js, exactly because I had issues with `content`.